Library/Formula/cfssl.rb

```diff
@@ -0,0 +1,103 @@
+require "language/go"
+
+class Cfssl < Formula
+  desc "CloudFlare's PKI toolkit"
+  homepage "https://cfssl.org/"
+  url "https://github.com/cloudflare/cfssl/archive/1.1.0.tar.gz"
+  sha256 "589222d922aa4ebae2039a54c46b9a7c2df4c95d78026ab35768a6cf62836ca9"
+  head "https://github.com/cloudflare/cfssl.git"
+
+  bottle do
+    cellar :any_skip_relocation
+  end
+
+  option "with-pkcs11", "Enables PKCS11 support and adds dependency on libtool"
```

Comment:
I left this as an option, because pkcs11 is binary api. But, guess it would not make lot of harm, if it's on by default. miegk/pkcs11 loads pkcs11 drivers dynamically anyway.